compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/text/selection/KeyboardActions.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.text.selection
+
+import androidx.compose.ui.input.key.Key
+import androidx.compose.ui.test.KeyInjectionScope
+import androidx.compose.ui.test.pressKey
+
+interface KeyboardActions {
+    fun KeyInjectionScope.selectAll()
+    fun KeyInjectionScope.selectLineStart()
+    fun KeyInjectionScope.selectTextStart()
+    fun KeyInjectionScope.selectLineEnd()
+    fun KeyInjectionScope.selectTextEnd()
+    fun KeyInjectionScope.deleteAll()
+}
+
+object DefaultKeyboardActions : KeyboardActions {
```

Comment:
Why "Default" but not "Windows"?

Reply:
Throughout all our code base all related entities have default in their name. I guess it's because the default one is not necessarily the windows one

Reply:
Asking because of this line:
```kt
    override fun toString() = "Win"
```

Reply:
I see. Yeah, that's a good point, the only reason was to have a 3-letter abbreviation (it is shown in test name). What dou you thing if we'll rename this to Dflt or something?

Reply:
Why should it be 3-letter?

My initial thought was to rename the class from Default* to Windows*